gui/src/components/preboarding/license.vue

```diff
@@ -53,12 +54,18 @@ import i18n from "./preboarding.json";
   }
 })
 export default class CortxLicense extends Vue {
+  public unsupportedFeatures = unsupportedFeatures;
   private data() {
     return {
       showLicenseAgreement: false,
       getStarted: false
     };
   }
+  
+  private mounted() {
```

Comment:
Is this method required? If not please remove it.

Reply:
Removed this too as updated router file.